nishimweelysee/Elysee-CapstoneProject
# Ft single blog page info

#### What does this PR do?
> This PR is for single blog information, when i click on any blog then be able to see more information on that blog.
>
#### Description of Task to be completed
>- user click to blog and directly go to single blog page
>- That page contains the information of that blog
>
#### How should this be manually tested?
> open website
> goto blog page
> click on any blog
> You will see it information page.
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
> FT (Single blog page information ) use this link to see my trello card :https://trello.com/c/QdMnVQVU
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/93688477-6bf66580-fac6-11ea-945f-d9dd09f02feb.png)
